keda/values.yaml

```diff
@@ -171,6 +172,7 @@ prometheus:
   operator:
     enabled: false
     port: 8080
+    portName: metrics
```

Comment:
As far as I see, this is not being used - Is this on purpose?

Reply:
Yeah. I left that out because I'm not sure if the operator metrics could also be exposed. From the docs it seemed like metrics-apiserver metrics is the only one needed: https://keda.sh/docs/2.1/operate/prometheus/

I can certainly add this to the `keda-deployment` config but then we would need to add a service to the keda operator in order to expose these. (afaik)

Reply:
Or I can remove `portName` and the related docs change for the time being @tomkerkhove. What would you suggest?

Reply:
I like the consistency, but if it's doesn't do anything I'd suggest to leave it out until we need it - What do you think?

Reply:
Exposing metrics from Operator would be nice to have as well. It doesn't have to be solved in this PR of course :)

Reply:
The whole prom metrics stack in KEDA (both Operator & Metrics Server) deserves some unification and refactoring.